### PR TITLE
Remove redux from ChatBubble

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Removes access to redux from ChatBubble. This simplifies the LevelDetailsDialog work as it limits the number of places that need to reason about redux. Again, all props accessed are page constants that can easily be passed around.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]

**Review by dmcavoy — APPROVED** [review]
Thanks for continuing to clean this up!

File: apps/src/templates/instructions/InlineFeedback.jsx [hunk 1]
```diff
@@ -11,7 +11,10 @@ class InlineFeedback extends Component {
     borderColor: PropTypes.string,
     extra: PropTypes.string,
     message: PropTypes.string.isRequired,
-    styles: PropTypes.object
+    styles: PropTypes.object,
+    isMinecraft: PropTypes.bool,
+    skinId: PropTypes.string,
+    textToSpeechEnabled: PropTypes.string
```
Comment (cforkish) [inline apps/src/templates/instructions/InlineFeedback.jsx:17]: @bethanyaconnor i assume this was a typo? we're passing in a bool and `ChatBubble` is expecting a bool, so i'm getting a "Failed prop type" warning from this. i'm going to fix it unless i'm misunderstanding somerthing.

Comment (bethanyaconnor) [reply apps/src/templates/instructions/InlineFeedback.jsx:17]: Yes, but it should've been fixed in https://github.com/code-dot-org/code-dot-org/pull/39618. Is it still happening?

Comment (cforkish) [reply apps/src/templates/instructions/InlineFeedback.jsx:17]: ah my bad, i haven't rebased onto staging in a while. thanks!!

**Pull request merged**